src/pages/index.tsx

```diff
@@ -1,13 +1,6 @@
-/** @jsxImportSource @emotion/react */
-
-import { Localized } from '@fluent/react'
-import Layout from '../components/layout'
-import SEO from '../components/seo'
-
-const IndexPage = () =>
-	<Layout>
-		<SEO title="Home" />
-		<Localized id="hello"><h1>Hello world!</h1></Localized>
-	</Layout>
-
-export default IndexPage
+import { EFRouter } from "../navigation/router";
+
+export default EFRouter;
+
+export * as register from "./register";
+export * as four_0_four from "./404";

```

Comment:
I'm a bit confused what these are for, since Gatsby would just load the default export, right?

Reply:
Ahhh, I just forgot to remove those 😅